app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1 class="m-2"> Add new expense </h1>
+<%= form_with(model: [ @group, @expense], local: true) do |f| %>
+  <div class="f-group p-2 m-2" >
+    <%= f.label :name, class: "control-label" %>
+    <%= f.text_field :name, class: "form-control", placeholder: "expense name", style: "width: 300px; border: 1px solid #000;" %>
+  </div>
+
+  <div class="form-group p-2 m-2">
+    <%= f.label :amount, class: "control-label" %>
+    <%= f.number_field :amount, class: "form-control", placeholder: "amount", style: "width: 300px; border: 1px solid #000;" %>
+  </div>
+
+  <div class="form-group p-2 m-2" style="margin-top: 1rem;">
+    <%= f.label :group, class: "control-label" %>
+    <%= f.collection_select :group, @groups, :name, :name, prompt: "Select a group", class: "form-control", required: true %>
+  </div>
+  
+  <br />
+
+  <div class="p-2 d-flex justify-content-between" style="background-color: #5fb523; position: fixed; bottom: 0; width: 100vw;">
+    <div style="margin-right: 2rem">
+      <%= f.submit "Create expense", class: "btn btn-outline-light" %>
+    </div>
+    <div>
+      <%= link_to "Back to Expenses", group_expenses_path(group_id: params[:group_id]), class: "btn btn-outline-light" %>
+    </div>
+  </div>
+<% end %>
```

Comment:
- [ ] I understand that programmers love being creative with designs. However, it is paramount that designers stick to design guidelines in projects. This shows utmost professionalism and attention to detail. On this page, I would recommend you do the following ✍️,

- Move the **_Back to groups button_** to the top of the page in the navigation section (You don't need to change it to an icon, just position it at the right corner of the navigation section. 😃 
- Make the _**Create expense**_ look identical to that of the Behance design layout for buttons on their templates. 🖥️  

--- 

| Your Design  | Behance Design |
| ------------- | ------------- |
| ![image](https://github.com/DevaWinner/expenditure/assets/77106962/0e19a473-968f-4a1d-94a5-7b2b2a916e75)  | ![image](https://github.com/DevaWinner/expenditure/assets/77106962/e7ad23e4-29d8-42f7-8551-8d0be3820389)  |

Reply:
I have restructured app interface

Reply:
Fixed.